tests/e2e/test_driver.py

```diff
@@ -101,6 +101,60 @@ def assertEqualRowValues(self, actual, expected):
         for act, exp in zip(actual, expected):
             self.assertSequenceEqual(act, exp)
 
+class PySQLPytestTestCase():
```

Comment:
This is a duplicate of `PySQLTestCase` that doesn't subclass `unittest.TestCase`. This is necessary if we want to use `pytest.mark.parametrize` since that decorator fails when applied to tests contained in a subclass of `unittest.TestCase`.